ui/DesignSystem/Component/SourceBrowser/FileSource.svelte

```diff
@@ -67,7 +80,10 @@
   <header>
     <div class="file-header">
       <Icon.File />
-      <span class="file-name">{path}</span>
+      <span class="file-name">
+        <a href={projectSource(projectId)} use:link>{projectName}</a>
```

Comment:
Not ideal if deeply nested components have knowledge about something like where to link to, makes them pretty hard to reuse.